.github/CODEOWNERS

```diff
@@ -565,22 +572,23 @@ x-pack/examples/files_example @elastic/kibana-app-services
 /x-pack/plugins/security_solution/public/common/components/threat_match @elastic/security-solution-platform
 
 ## Security Solution cross teams ownership
-/x-pack/plugins/security_solution/cypress/downloads @elastic/security-detections-response @elastic/security-threat-hunting
 /x-pack/plugins/security_solution/cypress/fixtures @elastic/security-detections-response @elastic/security-threat-hunting
 /x-pack/plugins/security_solution/cypress/helpers @elastic/security-detections-response @elastic/security-threat-hunting
+/x-pack/plugins/security_solution/cypress/integration/detection_rules @elastic/security-detections-response-rules @elastic/security-detections-response-alerts
```

Comment:
@marshallmain Do you agree with shared ownership of this one?

Reply:
Shared ownership requires approval from both teams, right? I feel like for cypress tests approval from either team (or any detections and response team) would be sufficient.

Reply:
@marshallmain I'm not entirely sure, but it seems like the [docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file) say that it's _either_, not both:

```
# In this example, any change inside the `/scripts` directory
# will require approval from @doctocat or @octocat.
/scripts/ @doctocat @octocat
```